### PR TITLE
First steps of My portfolio project: toolbar and headline

### PR DESCRIPTION
**Pull request by CindyDorantes**
First review request included:
- Add index.html file with 2 sections: toolbar and headline
- Add style.css file for styling according to template 3 (available in Figma)
- Add README file
- Install and run linters 

Second review request. Changes included:
- Modify tags for logo, hamburger, and paragraph
- Remove <nav> tag from the unsorted list <ul>

### CODE REVIEW

File: Index.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    <link rel="stylesheet" href="style.css">
+    <!-- To embed a font -->
+    <style>
+        @import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;600;700&display=swap');
+    </style>
+    <!-- Font 'Inter' embeded-->
+</head>
+<body>
+    <header class="toolbar">
+        <span class="welcome">Welcome</span>
```
Comment (ShahierNashaat) [inline Index.html:17]: - [ ] Kindly be noticed that the logo should be inside a link (`a` tag).

File: Index.html [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        <span class="hamburguer"><img src="./Images/Union.png" alt="hamburguer menu"></span>
```
Comment (ShahierNashaat) [inline Index.html:18]: - [ ] Kindly be noticed that the hamburger button should be inside a `nav` tag.

File: Index.html [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+        
+    </header>
+    <section class="headline">
+        <h1>
+            Hey there. I'm Cindy
+            <br>
+            <span class="greenheader">I'm a software developer</span>
+        </h1>
```
Comment (ShahierNashaat) [inline Index.html:26]: - [ ] Kindly make the title in an `h1` tag and the subtitle in an `hx` tag (examples: `<h1>,<h2>,<h3>,` etc...).
- [ ] so for example:-
```
<h1>Hey there. I'm Cindy</h1>
<h2>I'm a software developer</h2>
```

File: Index.html [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+        <h2>
+            I can help you build a product, feature or website. Look through some of my work and experience! 
+            If you like what you see and have a project you need coded, don't hesitate to contact me.
+        </h2>
```
Comment (ShahierNashaat) [inline Index.html:30]: - [ ] Kindly be noticed that you should put this paragraph in a `p` tag.

File: Index.html [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+        <nav>
+            <ul>
+                <li><a href="https://twitter.com/CindyDorantes10"><img src="./Images/twitter.png" alt="Twitter icon" class="socialicon"></a></li>
+                <li><a href="https://www.linkedin.com/in/cindydorantessanchez/"><img src="./Images/Linkedin_icon.png" alt="LinkedIn icon" class="socialicon"></a></li>
+                <li><a href="https://medium.com/"><img src="./Images/medium.png" alt="Medium icon" class="socialicon"></a></li>
+                <li><a href="https://github.com/CindyDorantes"><img src="./Images/github.png" alt="GitHub icon" class="socialicon"></a></li>
+                <li><a href="https://angel.co/"><img src="./Images/angellist.png" alt="AngelList icon" class="socialicon"></a></li>
+            </ul>
+        </nav>
```
Comment (ShahierNashaat) [inline Index.html:39]: - [ ] Kindly remove the `nav` tag from this section as it shouldn't be here.

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi @CindyDorantes , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rukundoeric — APPROVED** [review]
Hi @CindyDorantes,

_Excellent job on this project. Keep it up 🎆_

_Congratulations! 🎉  on the splendid job 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_

## Highlights
  - Follows git workflow.  💯
  - Codebase looks clean and professional. 💯 


Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

------
>As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**